model-optimizer/mo/moc_frontend/pipeline.py

```diff
@@ -96,4 +97,8 @@ def shape_to_array(shape: PartialShape):
             input_model.set_partial_shape(place, new_partial_shape)
 
     ngraph_function = moc_front_end.convert(input_model)
+
+    # Apply preprocessing (mean/scale/reverse_channels/convert_layout/etc)
+    apply_preprocessing(ov_function=ngraph_function, argv=argv)
```

Comment:
I propose to move a call of `apply_preprocessing` to `model-optimizer\mo\moc_frontend\serialize.py` in order to have all transformations in one place: preprocessing, user transforms, fp16 compression.

Reply:
Done